test/TestHelper.js

```diff
@@ -67,5 +67,10 @@ export async function expectNoViolations(node, options = {}) {
   });
 
   expect(results.passes).to.be.not.empty;
+
+  if (results.violations.length) {
+    console.log(JSON.stringify(results.violations, null, 2));
+  }
```

Comment:
👍

Reply:
Why the stringification? Wouldn't it be enough to inspect it in the browser dev tools?

Reply:
Well, this would be visible in CI for example, and also for running all tests I find the console is a bit easier as it doesn't flood with text unlike karma in the browser.